starter-core/src/main/java/io/micronaut/starter/feature/build/maven/templates/pom.rocker.raw

```diff
@@ -148,38 +148,12 @@ MavenBuild mavenBuild
 @dependency.template("io.micronaut", "micronaut-http-client", "test", null, false, null)
 }
 @if (features.testFramework().isJunit()) {
-@dependency.template("org.junit.jupiter", "junit-jupiter-api", "test", null, false, null)
-@dependency.template("org.junit.jupiter", "junit-jupiter-engine", "test", null, false, null)
-@dependency.template("io.micronaut.test", "micronaut-test-junit5", "test", null, false, null)
     @if (features.contains("hamcrest")) {
 @dependency.template("org.hamcrest", "hamcrest", "test", null, false, null)
     }
     @if (features.contains("mockito")) {
 @dependency.template("org.mockito", "mockito-core", "test", null, false, null)
     }
```

Comment:
Why not moving these too? Then you could get rid of the whole `@if (features.testFramework().isJunit()) {}` block

Reply:
Oh, never mind, I have just come across #1428